src/routes.tsx

```diff
@@ -82,6 +85,24 @@ export const routes = [
   {
     path: "liquidity",
     element: <PoolsPage />,
+    children: [
+      {
+        path: "/",
+        element: <Navigate to="allPools" />,
+      },
+      {
+        path: "myLiquidity",
+        element: <MyLiquidity />,
+      },
+      {
+        path: "allPools",
```

Comment:
I would keep paths consitent with other paths. Lowercase and dash separated.
* all-pools
* my-liquidity